docs/builder.rst

```diff
@@ -290,11 +341,20 @@ To include the source as a content-addressed URI, ``Py-EthPM`` can pin your sour
    ...     "./Owned.sol": "ipfs://Qme4otpS88NV8yQi8TfTP89EsQC5bko3F5N1yhRoi6cwGV"
    ...   }
    ... }
+   >>> # With `pin_source()`
    >>> built_manifest = build(
    ...     BASE_MANIFEST,
    ...     pin_source("Owned", compiler_output, ipfs_backend, package_root_dir=owned_dir),
    ... )
    >>> assert expected_manifest == built_manifest
+   >>> # With `pin_sources()` for multiple sources from the same compiler output
+   >>> pinner = pin_sources(compiler_output, ipfs_backend, package_root_dir=owned_dir)
```

Comment:
Maybe to avoid visual confusion between `pin_source` and `pin_sources` since they only differ by the trailing `s` we could rename `pin_sources -> source_pinner` which aligns better with the naming convention we are setting up here in the docs.